app/views/taxons/deletion.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>You are about to delete "<%= taxon.title %>"</h1>
+
+<%= render partial: 'deletion_warning',
+  locals: {
+    taxon: taxon,
+    children: children,
+    tagged: tagged } %>
+
+<% if children.any? %>
+  <%= render partial: 'children', locals: { children: children } %>
+<% end %>
+
+<% if tagged.any? %>
+  <%= render partial: 'tagged_content', locals: { tagged: tagged } %>
```

Comment:
We could also use this in [taxons#show](https://github.com/alphagov/content-tagger/blob/master/app/views/taxons/show.html.erb#L36-L56) and remove some code duplication, what do you think?